nbformat/_version.py

```diff
@@ -1,9 +1,39 @@
-# Make sure to update package.json, too!
-version_info = (5, 6, 0, "b0")
+# Use "hatchling version xx.yy.zz" to handle version changes
+import re
 
-if len(version_info) <= 3:
-    version_extra = ""
-else:
-    version_extra = version_info[3]
+try:
+    from importlib.metadata import version
+except ImportError:
+    # Not available until Python 3.8
+    version = lambda _: "0.0.0"  # noqa
```

Comment:
looks like we need `importlib_metadata` as a fallback, and probably an <py38 dep for #306

Reply:
Done!